pkg/api/validate/dynamic/dynamic.go

```diff
@@ -55,9 +59,12 @@ type Dynamic interface {
 
 type dynamic struct {
 	log            *logrus.Entry
+	authorizer     refreshable.Authorizer
```

Comment:
@mbarnes, will this need to be updated once #2667 is merged?

Reply:
I think this can be removed.  It's probably from before @mbarnes's PR.  I can see that there's no need to store this into dynamic.

Reply:
I removed it in my latest squashed push.  I believe this one is no longer needed so I removed it.  But apart from here, I changed all of refreshable to autorest accordingly.